src/coreclr/inc/readytoruninstructionset.h

```diff
@@ -67,6 +67,7 @@ enum ReadyToRunInstructionSet
     READYTORUN_INSTRUCTION_Zba=57,
     READYTORUN_INSTRUCTION_Zbb=58,
     READYTORUN_INSTRUCTION_Sve2=59,
+    READYTORUN_INSTRUCTION_Pac=60,
```

Comment:
I do not think it makes sense to add Pac to the ReadyToRunInstructionSet. It would mean that binaries generated with Pac would be rejected on machines without Pac. We do not want that. We want the same binary with Pac enabled to work both on machines with Pac and without Pac.

Pac is not like the instruction sets.